projects/unrar/unrar_fuzzer.cc

```diff
@@ -1,12 +1,32 @@
+#include <stddef.h>
+#include <stdint.h>
+
+#include <filesystem>
 #include <fstream>
 #include <memory>
-#include <sstream>
 #include <string>
+#include <system_error>
 #include <unistd.h>
 
 #include "rar.hpp"
 
+namespace fs = std::__fs::filesystem;
+
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
+  // unrar likes to create files in the current directory.
```

Comment:
How slow is the fuzzer from doing this now? Maybe just create an object that creates/cds into the dir and who's destructor destroys it?

Reply:
On my machine, with this change:
```
#3110033        REDUCE cov: 1425 ft: 4502 corp: 1001/70Kb lim: 254 exec/s: 1477 rss: 56Mb L: 58/254 MS: 1 EraseBytes-                                                                  
```

Reply:
As discussed offline, it's probably OK to go ahead as-is for now.

Reply:
For posterity: We discussed creating a `static` object whose destructor cleans up the files and the temporary directory, but it the problem with that approach is that the destructor may not be called in the case of a crash so the files won't be cleaned up.